discussion/src/test/java/org/openedx/discussion/presentation/comments/DiscussionCommentsViewModelTest.kt

```diff
@@ -94,7 +94,8 @@ class DiscussionCommentsViewModelTest {
         mapOf(),
         0,
         false,
-        false
+        false,
```

Comment:
IMO, it is better to `Add names to call arguments` in this file. Also, remove other warnings.